src/status_im2/common/standard_auth/keycard/view.cljs

```diff
@@ -0,0 +1,178 @@
+(ns status-im2.common.standard-auth.keycard.view
```

Comment:
Nothing in this namespace has usages in the repo, so how can we know the code works and the UI is correct according to dev & designer review?

The acceptance criteria in the issue says the components should be used/tested in preview namespaces. I disagree with that, these are not quo components and shouldn't live inside `src/status_im2/contexts/quo_previews/`.

Reply:
@ilmotta @J-Son89  I added previews for my changes to make them visible at least for now.

Reply:
Thanks @hknakn!